.asf.yaml

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This file configures github and jira notifications based on
+# https://s.apache.org/asfyaml-notify
+
+github:
+  description: "Apache Avro is a data serialization system."
+  homepage: https://avro.apache.org/
+  labels:
+    - avro
+    - bigdata
+    - c
+    - c#
+    - c++
+    - csharp
+    - dotnet
+    - java
+    - .net
+    - perl
+    - php
+    - python
+    - ruby
+
+  enabled_merge_buttons:
+    squash: false
```

Comment:
I always use squash and merge, what do other use?

Reply:
Well spotted!  I largely prefer squash-and-merge in 99% of the cases, and rarely (but occasionally) rebase.

Reply:
Ah good point, I don't know why I assumed that squash and merge created an additional merge commit but it does not seem to be the case
https://help.github.com/en/github/administering-a-repository/about-merge-methods-on-github

I will update this to let the two options that do not create the additional commit, well spotted @Fokko